### ISSUE TITLE
Showing internal errors on missing commands

### ISSUE THREAD

**Issue by jorgemoralespou** [minishift/minishift#370]
It would be nice if the error message didn't show the internal log

````
✘ jmorales:~ ➭ minishift destroy
Error: unknown command "destroy" for "minishift"
Run 'minishift --help' for usage.
F0206 11:47:37.816139    5297 root.go:95] unknown command "destroy" for "minishift"
````

**Comment by praveenkumar** [comment]
@jorgemoralespou I checked with minikube also same and it also show internal log, I did some reading around cobra and looks like we don't need to exit with err mentioned because other part of code anyway have proper exit functionality.

As per cobra doc : https://github.com/spf13/cobra#suggestions-when-unknown-command-happens

```
 ./minikube adfdad
Error: unknown command "adfdad" for "minikube"
Run 'minikube --help' for usage.
F0207 17:39:42.018486   20150 root.go:108] unknown command "adfdad" for "minikube"
``` 

Changes which I think from our side is just https://github.com/minishift/minishift/blob/master/cmd/minikube/cmd/root.go#L95 change it to, wdyt?

```
os.exit(-1)
```

**Comment by jorgemoralespou** [comment]
@praveenkumar I'm no engineer, don't know go, so ask @hferentschik  :-D

**Comment by praveenkumar** [comment]
@jimmidyson / @hferentschik  what you folks think about https://github.com/minishift/minishift/issues/370#issuecomment-277982017 ? Does that make sense?

**Comment by jimmidyson** [comment]
@praveenkumar Yeah that sounds good to me. It's what OpenShift does too btw - https://github.com/openshift/origin/blob/master/cmd/oc/oc.go#L36-L38.

**Comment by hferentschik** [comment]
Merged via pull request #383
